dist/README.md

```diff
@@ -0,0 +1,27 @@
+# Installing
+
+Pick the binary that matches your operating system and architecture. For
+example, on my Macbook Pro I would use `ftree-darwin-amd64`. Copy that somewhere
+in your path and rename to `ftree`. Presto bango.
+
+## Verifying
+
+The best way to verify the release is to examine the GPG signature. You can find
+my public key at http://calebthompson.io/pubkey.asc,
+<https://pgp.mit.edu/pks/lookup?op=get&search=0x1621ADC2A0ACE70A>, and via GnuPG
+with `gpg --recv-keys A0ACE70A`.
+
+You can find a signed and verifiable statement of my ownership of that key at
+<https://github.com/calebthompson/i-am>.
+
+Once you have that key, verification is pretty easy:
+
+```shell
+$ gpg --verify ftree-<os>-<arch>.sig ftree-<os>-<arch>
+```
+
+If you don't have GnuPG, you can verify the SHA sums.
+
+```shell
+$ shasum --algorithm 256 --check ftree-<os>-<arch>.sha256
```

Comment:
Another thing I can do here is store all checksums together and they can be verified like:

```                                                   
❯ shasum -a 256 --check checksums 
ftree-darwin-386: OK
ftree-darwin-amd64: OK
ftree-freebsd-386: OK
ftree-freebsd-amd64: OK
ftree-freebsd-arm: OK
ftree-linux-386: OK
ftree-linux-amd64: OK
ftree-linux-arm: OK
ftree-openbsd-386: OK
ftree-openbsd-amd64: OK
ftree-windows-386: OK
ftree-windows-amd64: OK
```